lib/google/gax/version.rb

```diff
@@ -29,6 +29,6 @@
 
 module Google
   module Gax
-    VERSION = '0.8.0'.freeze
+    VERSION = '0.9.0'.freeze
```

Comment:
Can this be 0.8.1 instead of 0.9.0? It looks to be backwards compatible and I'd like to not have to release new versions of the google-cloud-* gems again